### PR TITLE
add readme

### PR DESCRIPTION
**Pull request by Keyshal-211**
## Description of changes
Please provide a short description of the change or if the change is sufficiently described in the GitHub issue, provide a link to the issue.

  
## How has this been tested?

Please describe the test strategy you followed. If not tested, please explain why.

- [ ] automated unit test
- [ ] automated integration test
- [ ] manual, exploratory test

In case of manual test, please document the test well including a set of user instructions and prerequisites. Each including an action, it's result, and where appropriate a screenshot.
    
## Checklist before requesting a review

- [X] My code is easy to understand
- [x] My code follows the [coding guidelines](https://github.com/IT-REX-Platform/wiki/blob/main/dev-manuals/backend/coding-guidelines.md) of this project
- [ ] My code fulfills all acceptance criteria
- [ ] The test coverage (line and branch) is reasonably high, especially on Service classes and other classes with complex logic
- [x] I have made corresponding changes to the documentation or
  the [wiki/adr](https://github.com/IT-REX-Platform/wiki/tree/main/adr)
- [ ] I made no breaking changes in the database schema or if so, I will perform a database migration

## Checklist for reviewer

- The code is easy to understand
- The code follows
  the [coding guidelines](https://github.com/IT-REX-Platform/wiki/blob/main/dev-manuals/backend/coding-guidelines.md) of
  this project
- The code is tested or if not, the reason is documented or discussed
- The added and existing tests reasonably cover the code change
- The code has no breaking changes in the database schema or if so, the assignee is aware of it and
  will perform a database migration

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,13 +1,9 @@
 # Course Service
 
-## Description
+The Course Service plays a pivotal role in the platform's educational ecosystem by serving as a comprehensive repository of essential course-related information. This information encompasses a wide array of details, including course names, comprehensive course descriptions, and the structural breakdown of courses into chapters and sections. This wealth of data is pivotal in guiding both educators and students through their learning journey within the platform.
```
Comment (PaulBredl) [inline README.md:3]: The breakdown into section does not happen in the course service
Also I would not use the word ecosystem... idk seems like overselling our system

**Comment by PaulBredl** [comment]
@Keyshal-211 I didn't know that this service already had a description. Honestly, I think the old one is more accurate. I think you should rather update the old description and remove the sentence that it invoked keycloak, because that is outdated

File: README.md [hunk 2]
```diff
@@ -1,13 +1,10 @@
 # Course Service
 
-## Description
+The Course Service serves as a pivotal component within our system, offering essential features related to courses and chapters:
```
Comment (PaulBredl) [inline README.md:3]: Remove "pivotal"  and "essential", imo
```suggestion
The Course Service offers features related to courses and chapters:
```

Comment (Keyshal-211) [reply README.md]: done

**Review by PaulBredl — APPROVED** [review]
@Keyshal-211 you can merge if you want

**Pull request merged**